student.rb

```diff
@@ -16,3 +16,14 @@ def play_hooky
     "¯\(ツ)/¯"
   end
 end
+
+def create_student(people)
+  print 'Age:'
+  age = gets.chomp
+  print 'Name:'
+  name = gets.chomp
+  print 'Has parent permission?[Y/N]:'
+  permission = gets.chomp
+  people << Student.new(age, name, parent_permission: permission.upcase == 'Y')
+  puts 'Student created successfully'
+end
```

Comment:
- [ ] As I mentioned above ⬆ All requests for inputs should have been placed in classes or methods whose responsibilities are to request input for reasons outlined above. 👌

> For Example 💡
```
// require_inputs.rb

def grap_student_data
  print 'Age:'
  age = gets.chomp
  print 'Name:'
  name = gets.chomp
  print 'Has parent permission?[Y/N]:'
  permission = gets.chomp

  [age, name, permission]
end
```
```
// student.rb

def  create_student(people)
   age, name, permission = grap_student_data
  people << Student.new(age, name, parent_permission: permission.upcase == 'Y')
  puts 'Student created successfully'
end
```